Makefile

```diff
@@ -100,8 +100,15 @@ integration:
 	./hack/test-integration.sh
 
 .PHONY: e2e
-e2e:
+e2e: bin
+ifeq ($(RUNNER),kops)
 	./hack/e2e/run.sh
+else ifeq ($(RUNNER),kind)
+	./hack/start-dev-env-dynamicfile.sh
```

Comment:
Nit: so the "dev env" for dynamic file inculdes an automatic e2e test run, it would be nice if the UX was the same w.r.t the other dev env scripts.  I think I probably would have preferred modifying the other dev scripts so that there is just one way of creating a dev environment and some configuration options which determine whether or not dynamic file backend mode is used.  But we can clean up this experience as part of a future change.